packages/delivery-expo/test/delivery.test.js

```diff
@@ -202,6 +202,40 @@ describe('delivery: expo', () => {
     })
   })
 
+  it('does not attempt to re-send oversized payloads', done => {
+    // A 401 is considered retryable but this will be override by the payload size check
+    const { requests, server } = mockServer(401)
+    server.listen(err => {
+      expect(err).toBeUndefined()
+
+      const lotsOfEvents = []
+      while (JSON.stringify(lotsOfEvents).length < 10e5) {
+        lotsOfEvents.push({ errors: [{ errorClass: 'Error', errorMessage: 'long repetitive string'.repeat(1000) }] })
+      }
+      const payload = {
+        events: lotsOfEvents
+      }
+
+      const config = {
+        apiKey: 'aaaaaaaa',
+        endpoints: { notify: `http://0.0.0.0:${server.address().port}/notify/` },
+        redactedKeys: []
+      }
+
+      let didLog = false
+      const log = () => { didLog = true }
+
+      delivery({ _config: config, _logger: { error: log, info: () => {} } }, fetch).sendEvent(payload, (err) => {
+        expect(didLog).toBe(true)
+        expect(enqueueSpy).not.toHaveBeenCalled()
```

Comment:
tests are going to fail here until https://github.com/bugsnag/bugsnag-js/pull/1823/files#diff-b6197bd8422edfa11579021015a96f02f31bc20634ad051b821c01e4ce178afdL18 lands in the version of `@bugsnag/core` used by bugsnag exp